src/imitation/envs/examples/model_envs.py

```diff
@@ -67,7 +67,7 @@ def make_obs_mat(n_states, is_random, obs_dim, rand_state=np.random):
     assert (
         obs_mat.ndim == 2 and obs_mat.shape[:1] == (n_states,) and obs_mat.shape[1] > 0
     )
-    return obs_mat
```

Comment:
Test suite errors because we added to `seals` a check for `dtype`. This is a consequence of upgrading the `seals` version to the GitHub master not rendering per se. I could split this into a separate PR if desired but it only affects a couple of places.